src/soc/ibm/power9/istep_9_7.c

```diff
@@ -0,0 +1,94 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+
+#include <cpu/power/istep_9.h>
+
+#include <console/console.h>
+#include <cpu/power/scom.h>
+#include <delay.h>
+#include <stdbool.h>
+
+#include "xbus.h"
+
+static void p9_fab_iovalid_link_validate(uint8_t chip)
+{
+	enum {
+		XBUS_1_LL1_LL1_LL1_IOEL_FIR_REG = 0x06011C00,
```

Comment:
Do we need so many LL1s? This is just `PB.IOE.LL1.IOEL_FIR_REG` in documentation so one LL1 should be enough.

Reply:
I preserved weird naming from `p9_xbus_scom_addresses.H`, can be simplified.